content/en/load-testing/Quick Start/40-run-a-test.md

```diff
@@ -31,11 +31,11 @@ com.xceptance.xlt.loadtests.TVisit.users = 1
 com.xceptance.xlt.loadtests.TVisit.arrivalRate = 500
 ```
 
-In this case we run a test for 1 hour with a 5 minute ramp-up period, and a 1 minute shutdown phase. The shutdown phase will not turn up in our measurements, but ramp-up does. You can exclude it later if desired. Total runtime is 1h 5min (measurement period + shutdown period).
+In this case we run a test for 1 hour with a 5 minute [ramp-up](../../glossary/#ramp-up-period-xlt) period, and a 1 minute [shutdown](../../glossary/#shutdown-period-xlt) period. The shutdown phase will not turn up in our measurements, but ramp-up does. You can exclude it later if desired. Total runtime is 1h 5min ( [measurement](../../glossary/#measurement-period-xlt) period + shutdown period).
```

Comment:
As the Glossary is for "ramp-up _period_", let's link the whole `[ramp-up period]`. Same for shutdown/measurement.